Assignment 1/src/com/company/Main.java

```diff
@@ -0,0 +1,129 @@
+package com.company;
+import java.io.*;
+import java.net.URL;
+import java.util.*;
+import java.util.stream.Collectors;
+
+class Main {
+
+    public static void main(String args[])  {
+        try {
+            String URLS_FILE_NAME = "urls.txt";
+            String WORDS_FILE_NAME = "words.txt";
+
+            File urlsFile = new File(URLS_FILE_NAME);
+            if (urlsFile.exists() == false) {
+                System.out.println("Please Create File for urls");
+                return;
+            }
+            RandomAccessFile urlsRandomAccessFile = new RandomAccessFile(urlsFile, "rw");
+            if (urlsRandomAccessFile.length() == 0) {
+                urlsRandomAccessFile.close();
+                System.out.println("Please write urls in file");
+                return;
+            }
+            File wordsFile = new File(WORDS_FILE_NAME);
+            if (wordsFile.exists() == false) {
+                System.out.println("Please Create File for words");
+                return;
+            }
+            RandomAccessFile wordsRandomAccessFile = new RandomAccessFile(wordsFile, "rw");
+            if (wordsRandomAccessFile.length() == 0) {
+                wordsRandomAccessFile.close();
+                System.out.println("Please write words in file");
+                return;
+            }
+            Set<String> wordsSet = new HashSet<String>();
+            while (wordsRandomAccessFile.getFilePointer() < wordsRandomAccessFile.length()) {
+                wordsSet.add(wordsRandomAccessFile.readLine());
+            }
+            String urlString;
+
+            Map<String, Integer> hashMap = new HashMap<String, Integer>();
+            System.out.println("=================================");
+            while (urlsRandomAccessFile.getFilePointer() < urlsRandomAccessFile.length()) {
+                StringBuffer sb = new StringBuffer();
+                urlString = urlsRandomAccessFile.readLine();
+                URL url = new URL(urlString);
+                Scanner sc = new Scanner(url.openStream());
+                System.out.println(urlString);
+                while (sc.hasNext()) {
+                    sb.append(sc.next());
+                    sb.append("  ");
+                    //System.out.println(sc.next());
+                }
+
+                String result = sb.toString();
+
+                result = result.replaceAll("<[^>]*>", "");
+                // System.out.println(result);
+                Map<String, Integer> individualHashMap = new HashMap<String, Integer>();
+                Map<String, Integer> wordsHashMap = new HashMap<>(); // this map only contains that words which is present in text file
+
+                String[] words = result.split(" ");
+
```

Comment:
can you please remove all the empty lines so the code should be more readable

Reply:
ok sir